src/app/fetch-status.service.ts

```diff
@@ -76,19 +76,28 @@ export class FetchStatusService {
 
     const observables = mapArr.map((x) => {
       const statusSubject$ = x.subject;
-      return this.http.get(x.link, {
+      return this.http.head(x.link, {
```

Comment:
Please add a comment on startFetching function explaining about this behavior. Saying basically we want to send a HEAD request and assume that site owner implement a HEAD request returning similar metadata to GET request. However if HEAD request returns non-ok status, we will send a GET request to the same endpoint